staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers.go

```diff
@@ -98,6 +98,7 @@ func SerializeObject(mediaType string, encoder runtime.Encoder, hw http.Response
 		attribute.String("protocol", req.Proto),
 		attribute.String("mediaType", mediaType),
 		attribute.String("encoder", string(encoder.Identifier())))
+	req = req.WithContext(ctx)
```

Comment:
this one is not needed, is it? `req` is not used anywhere below.

Reply:
it is used in https://github.com/kubernetes/kubernetes/blob/99ff62e87a2d881f4839159d33519a49d6eee5d9/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers.go#L106